app/scripts/controllers/preferences-controller.ts

```diff
@@ -300,6 +302,10 @@ const controllerMetadata = {
     persist: true,
     anonymous: true,
   },
+  overrideContentSecurityPolicyHeader: {
+    persist: true,
+    anonymous: true,
```

Comment:
I'm new to this controller. What does the `anonymous` flag do?

Reply:
Any reason not to track it? I'm guessing `anonymous: true` means _don't_ send to sentry?

Reply:
Confusingly, `anonymous: true` means that it's already anonymized.

It makes a little more sense if you consider that you can also specify a function here to transform the state into something anonymized, e.g. `anonymous: (address) => hash(address)`

We do have an issue for renaming the field to something less confusing though 😅 Suggestions on which name to use are welcome: https://github.com/MetaMask/core/issues/643